composer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "ingenerator/scheduled-task-runner",
+    "type": "library",
+    "require": {
+        "php": "~8.2.0",
+        "ext-pcntl": "*",
+        "ext-pdo": "*",
+        "dragonmantank/cron-expression": "^3.3.2",
+        "ingenerator/php-utils": "^2.0",
+        "psr/log": "^1.0 || ^2.0 || ^3.0",
+        "symfony/lock": "^5.4 || ^6.0 || ^7.0",
+        "symfony/process": "^5.4 || ^6.0 || ^7.0"
```

Comment:
Worth being aware (again a thing I hit yesterday) that technically the 6.0 series isn't tested as we're only running builds with highest & lowest... It should be fine here & with these 2 packages but for other stuff it's potentially best to skip supporting intermediate releases unless we have to.

Reply:
Yeah I clocked that too. I did at least manually check that it does pass on 6 in dev at the moment. We're running versions 4, 5, and 6 in projects at the moment so just trying to maintain the broadest compatibility until we get some of those projects bumped up too. Then we might be able to stop supporting the lower / older versions

Reply:
Grand, makes sense